### ISSUE TITLE
can I deploy the RoITransformer on faster rcnn?

### ISSUE THREAD

**Issue by tongtyr** [dingjiansw101/RoITransformer_DOTA#11]
hello:
         I want to deploy the roitransformer on faster rcnn. Can someone help me?

**Comment by dingjiansw101** [comment]
Yes, of course you can. I have tried to apply RoITransformer on faster rcnn in mmdetection version. And it get even higher results. I will release the code later.

**Comment by oceanleftsea** [comment]
Waiting for your mmdetection version sincerely.
